Source/ARTRest.h

```diff
@@ -46,23 +83,56 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (BOOL)stats:(ARTPaginatedStatsCallback)callback;
 
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * Queries the REST `/stats` API and retrieves your application's usage statistics. Returns a `ARTPaginatedResult` object, containing an array of `ARTStats` objects. See the [Stats docs](https://ably.com/docs/general/statistics).
+ *
+ * @param query An `ARTStatsQuery` object.
+ * @param callback A callback for retriving an `ARTPaginatedResult` object with an array of `ARTStats` objects.
+ * @param errorPtr A reference to the `NSError` object where an error information will be saved in case of failure.
+ *
+ * @return In case of failure returns false and the error information can be retrived via the `error` parameter.
+ * END CANONICAL PROCESSED DOCSTRING
+ */
 - (BOOL)stats:(nullable ARTStatsQuery *)query
      callback:(ARTPaginatedStatsCallback)callback
         error:(NSError *_Nullable *_Nullable)errorPtr;
 
 #if TARGET_OS_IOS
+/**
+ * BEGIN CANONICAL PROCESSED DOCSTRING
+ * Retrieves an `ARTLocalDevice` object that represents the current state of the device as a target for push notifications.
```

Comment:
I think 'retrieves' is not appropriate for the property, even if it is read only and technically designed like a method.
```suggestion
 * An `ARTLocalDevice` object that represents the current state of the device as a target for push notifications.
```

Reply:
https://github.com/ably/canonical-api-reference-prototyping/issues/25#issue-1367687965